### ISSUE TITLE
no keypoints in the frame

### ISSUE THREAD

**Issue by simonetortorella** [zju3dv/EasyMocap#439]
I was testing the package with different types of videos. The entire structure fails if it does not detect at least one keypoint, such as when there is no subject in the video or when the subject temporarily leaves the camera's view. I tried modifying some lines, but it seems that not only the output videos but also the output data are corrupted. What changes should I make?

**Comment by chingswy** [comment]
Hello, you should make sure that all views have at least one people.

**Comment by simonetortorella** [comment]
Ok, but if the subject goes passe through the camera , like opening a door and e termini the room, easymocap does not work properly, crushes insrantaneously.

Inviato da Outlook per iOS<https://aka.ms/o0ukef>
________________________________
Da: Qing Shuai ***@***.***>
Inviato: Monday, November 18, 2024 5:04:30 AM
A: zju3dv/EasyMocap ***@***.***>
Cc: Simone Tortorella ***@***.***>; Author ***@***.***>
Oggetto: Re: [zju3dv/EasyMocap] no keypoints in the frame (Issue #439)


Hello, you should make sure that all views have at least one people.

**Comment by raflitp** [comment]
Of course, that's why you need a bunch of cameras if there's some props blocking one of the cameras. If there's no person it will crash obviously because there's no person to detect and predict

**Comment by simonetortorella** [comment]
Ok, but I expected that in a 30 second video in which the subject is visible for 10 seconds, the entire video would be analyzed with consequent restitution of the 30 seconds and rendering only when the keypoints are found (i.e. in the 10 seconds in which the subject is present), without giving an error from the beginning, as soon as the keypoints are not identified in frame 0. But if it's an easymocap specification, that's fine, thanks.
